sunpy/net/jsoc/jsoc.py

```diff
@@ -547,8 +547,8 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
 
         if urls:
             if progress:
-                print_message = "{0} URLs found for download. Full request totalling {1}MB"
-                print(print_message.format(len(urls), request._d['size']))
+                log_message = "{0} URLs found for download. Full request totalling {1}MB"
+                log.info(log_message.format(len(urls), request._d['size']))
```

Comment:
I changed this as we should really just be dumping to the logger instead of a print. But maybe others like it this way?

Reply:
I'm in favor of all of these messages being handled by the logger as it gives the user a standardized way to control the output.